src/guild_configs/lbp.js

```diff
@@ -0,0 +1,914 @@
+/// <reference path="../types.d.ts" />
+
+import Category from "../Category.js";
+import { MULTI_GAME } from "../misc.js";
+import * as misc from "../misc.js";
+import Race from "../Race.js";
+
+import Discord from "discord.js";
+import https from "https";
+
+const emotes = {
+	ppjE: "<:ppjE:230442929859198977>",
+	ppjSmug: "<:ppjSmug:230442929838227457>",
+	ppjWink: "<:ppjWink:367254404811980801>",
+	rbdBingo: "<:rbdBingo:394255134340677634>"
+};
+
+const ilAliases = [ "il", "ils" ];
+const anyNCAliases = [ "anynooverlord", "nocreate", "nooverlord", "anync", "anyno", "nc", "no" ];
+
+const ilCategory = {
+	aliases: ilAliases,
+	il: true
+};
+
+const COOP_REGEX = /coop|[2-5]p(layers?)?/;
+const LIGHTHOUSE_REGEX = /[a-z\.]*\.[a-z]{2,4}\/slot\/[1-9]*/mi;
```

Comment:
Good (altho the flag `m` is useless and I think you should use `+` instead of `*`), but there should be a list of known domain names. I don't want people to have the power to let Bingo Bot download arbitrary websites.